operations/cancellationpolicies_new.md

```diff
@@ -0,0 +1,112 @@
+
+Note this operation uses [Pagination](../guidelines/pagination.md) and supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v2/cancellationPolicies/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "EnterpriseIds": [
+        "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "4d0201db-36f5-428b-8d11-4f0a65e960cc"
+    ],
+    "ServiceIds": [
+        "e654f217-d1b5-46be-a820-e93ba568dfac"
+    ],
+    "CancellationPolicyIds": [
+        "fe795f96-0b64-445b-89ed-c032563f2bac"
+    ],
+    "UpdatedUtc": {
+        "StartUtc": "2023-04-27T11:48:57Z",
+        "EndUtc": "2023-04-27T11:48:57Z",
+    },
+    "Limitation": { "Count": 10 }
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). If not specified, the operation returns data for all enterprises within scope of the Access Token. |
+| `ServiceIds` | array of string | required, max 1000 items | Unique identifiers of the [Service](services.md#service). |
+| `CancellationPolicyIds` | array of string | optional, max 1000 items | Unique identifiers of the [Cancellation Policy](#cancellationpolicy). Required if no other filter is provided. |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Cancellation Policy](#cancellation-policy) was updated. Required if no other filter is provided. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of cancellation policies returned. |
+
+### Response
+
+```javascript
+{
+    "CancellationPolicies": [
+        {
+            "Id": "769fc613-838f-41a7-ac2a-aff100c3189f",
+            "Name": "Name of the cancellation policy"
+            "CreatedUtc": "2023-04-27T11:48:57Z",
+            "UpdatedUtc": "2023-04-27T11:48:57Z",
+            "Applicability": "Creation",
+            "FeeExtent": [
+                "TimeUnits",
+                "Products"
+            ],
+            "ApplicabilityOffset": "0M0D0:0:0.0",
+            "FeeMaximumTimeUnits": 0,
+            "AbsoluteFee": {
+                "Currency": "EUR",
+                "Value": 15.00
+            },
+            "RelativeFee": 0.00000000,
+            "CurrentVersion": "3"
```

Comment:
We should prbably have a parameter to get all versions or only current

Reply:
If we make it param we could get rid of the history endpoint